tools/restored-receipts-verifier/src/main.rs

```diff
@@ -93,6 +93,7 @@ fn main() -> Result<()> {
             .apply_transactions(
                 shard_id,
                 chunk_extra.state_root(),
+                None,
```

Comment:
It's not great that we have to change a whole bunch of tests to add an extra `None`, feels similar to #4745

Reply:
I agree!